common/src/main/java/org/mcaccess/minecraftaccess/MainClass.java

```diff
@@ -170,13 +170,11 @@ public static void setScreenReader(ScreenReaderInterface screenReader) {
     }
 
     public static void speakWithNarrator(String text, boolean interrupt) {
+        if (Strings.isNotEmpty(text) && Minecraft.getInstance().isWindowActive()) {
```

Comment:
This condition is backwards.
```suggestion
        if (Strings.isEmpty(text) || !Minecraft.getInstance().isWindowActive()) {
```